### PR TITLE
Document resourceGroup Parameter for Azure Disk

### LINKED ISSUE #15741: AzureDisk storage class parameters are incomplete
**This is a Feature Request**

On https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk, the list of configuration properties for Azure Disks is incomplete. I found https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/driver-parameters.md for the new CSI driver and found the parameter `resourceGroup` documented there -- which was exactly what I was searching for. 

It seems that the "old" built-in AzureDisk provider supports this as well. I added that parameter to my storage class, and it created disks connected to that storage class in the specified resource group automatically. So the parameter is working, just not documented.

**What would you like to be added**

Something like this, in the existing AzureDisk section.

* resourceGroup:  specify the resource group in which azure disk will be created.Must be an existing resource group name. If unspecified, disk will be placed in the the same resource group as current k8s cluster

**Why is this needed**

Completeness of documentation

### PR DESCRIPTION
**Pull request by mcurtiscmu**
AzureDisk storage class parameters are incomplete.
Added in the resourceGroup parameter into the documentation per the description.

### CODE REVIEW

**Comment by mcurtiscmu** [comment]
AzureDisk storage class parameters are incomplete.
Added in the resourceGroup parameter into the documentation per the description.

fixes #15741

**Comment by mcurtiscmu** [comment]
I signed it

**Comment by mcurtiscmu** [comment]
I signed it

**Comment by mcurtiscmu** [comment]
I signed it

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -588,6 +588,7 @@ parameters:
   it must reside in the same resource group as the cluster, and `location` is
   ignored. If a storage account is not provided, a new storage account will be
   created in the same resource group as the cluster.
+  * `resourceGroup`: Specify the resource group in which the Azure disk will be created. It must be an existing resource group name. If it is unspecified, the disk will be placed in the the same resource group as the current Kubernetes cluster.
```
Comment (andyzhangx) [inline content/en/docs/concepts/storage/storage-classes.md]: pls move this `resourceGroup` parameter to `New Azure Disk Storage Class (starting from v1.7.2)` section, thanks.

Comment (andyzhangx) [reply content/en/docs/concepts/storage/storage-classes.md]: also remove the first space in this line, thanks.

File: content/en/docs/concepts/storage/storage-classes.md [hunk 2]
```diff
@@ -611,6 +611,7 @@ parameters:
   unmanaged disk in the same resource group as the cluster. When `kind` is 
   `managed`, all managed disks are created in the same resource group as 
   the cluster.
+* `resourceGroup`: Specify the resource group in which the Azure disk will be created. It must be an existing resource group name. If it is unspecified, the disk will be placed in the the same resource group as the current Kubernetes cluster.
```
Comment (andyzhangx) [inline content/en/docs/concepts/storage/storage-classes.md]: remove one `the`

Comment (andyzhangx) [inline content/en/docs/concepts/storage/storage-classes.md]: remove one `the`

**Pull request closed without merging**